wirecard/oxid-ee
# WDPD-33 Credit card configuration

### This PR

* adds Credit Card payment method configuration
* shows Credit Card payment method config information on the checkout page
* shows payment method logo on the checkout page
* adds possibility of localising the payment's logo

### How to Test

* Login to the admin and activate the plugin.
* Go to Payment methods and select "Wirecard Credit Card".
* You are able to setup all needed fields for Credit card
* As a customer: You can see in checkout process Credit Card payment method. When selecting, debug info of payment method fields is displayed

### Jira Links
* https://jira.parkside.at/browse/WDPD-33
